txclib/project.py

```diff
@@ -157,14 +165,20 @@ def getset_host_credentials(self, host, username=None, password=None,
             save = True
 
         if save:
+            self._update_file_host(host)
+            self.txrc.set(host, 'username', username)
+            self.txrc.set(host, 'password', password)
+            self.save()
+        return username, password
+
+    def _update_file_host(self, host):
```

Comment:
@Diontsoumas maybe add a docstring in this function?

Reply:
@SofiaMargariti Sure, also updating some tests that fail.

Reply:
I think you should include the arguments and the return value in the docstring